test/Confluent.Kafka.IntegrationTests/Tests/AdminClient_AlterConfigs.cs

```diff
@@ -28,13 +28,13 @@
 
 namespace Confluent.Kafka.IntegrationTests
 {
-    public static partial class Tests
+    public partial class Tests
```

Comment:
the move away from `static` classes allows for per-test lifecycle methods.

Reply:
I strongly suggest you add a mode for running multiple tests in parallel, which is not only a great way to speed up test runs, but also verifies that there are no concurrency issues between client instances.
The librdkafka integration tests will run (by default) 5 simultaneous tests.